.github/workflows/flaky-test-monitor.yml

```diff
@@ -83,18 +83,15 @@ jobs:
       matrix:
         include:
           - name: crypto
-            make1: -C crypto setup
-            make2: unittest
+            setup:
```

Comment:
We should have an explicit target to call in `crypto/Makefile` now that `setup` target is no longer there.
By leaving this blank, this will execute the first target `crypto/Makefile` which is `c-format`. Is this the intention?

Reply:
No it's not the intention. I didn't know the first target is run. I'll add a `noop` target as you suggested

Reply:
I'm noticing the setup is called using `make ${{ matrix.setup }}`, so I guess this is only involving `./Makefile` and not `crypto/Makefile`. I'll add the `noop` target in `./Makefile` then.

Reply:
https://github.com/onflow/flow-go/pull/4817/commits/6c34ae3fccb9a23887c95d790e141a7c7000ed3f